orchestrator-api/src/main/java/com/sequenceiq/cloudbreak/orchestrator/model/Node.java

```diff
@@ -9,6 +11,8 @@
 
     private final String instanceType;
 
+    private final CloudIdentityType cloudIdentityType;
```

Comment:
please check if we could solve this another way without adding this field to the node